xcm/xcm-builder/src/origin_conversion.rs

```diff
@@ -241,6 +245,37 @@ where
 	}
 }
 
+/// Convert a Tinkernet Multisig `MultiLocation` value into a `Signed` origin.
+pub struct TinkernetMultisigAsNative<RuntimeOrigin>(PhantomData<RuntimeOrigin>);
```

Comment:
Similarly, I think from an organization perspective, it is more correct for this configuration to be placed in the kusama runtime, as it is the only user of it. But I see that it is more "convenient" to do it like this.

Reply:
Kusama is not the only user of these converters, parachains will also import these into their xcm configurations from xcm-builder. I believe this also responds the previous comment.

Reply:
I think for this converter it is fine. But about the previous comment: the `TINKERNET_PARA_ID` is only valid on Kusama, right?